swap_meet/vendor.py

```diff
@@ -1,2 +1,88 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    '''
+    if passing an inventory into an instance of Vendor,
+    the inventory should be a list of instances of the
+    Item class or its subclasses
+    '''
+    def __init__(self, inventory=None):
+        if inventory is None:
+            inventory = []
+        else:
+            for item in inventory:
+                if not isinstance(item, Item):
+                    raise TypeError
```

Comment:
This is a great extension! I'm excited to see that you wrote tests for `TypeError` in your `Item` objects. I wonder what the tests for this `TypeError` case would be! :thinking: *pondering teacher pose*